pkg/archive/archive.go

```diff
@@ -1,4 +1,8 @@
+// Package archive handles making container tar archives
 package archive // import "github.com/docker/docker/pkg/archive"
+// BUG(justincormack): This package does not handle symlink escapes safely in
+// general, as we only use it in a chroot environment where symlink espcapes are not
+// possible. Be careful using this code in a different environment.
```

Comment:
Should this be added to the package description above, so that it shows in go doc?

<img width="539" alt="Screenshot 2020-12-01 at 17 05 55" src="https://user-images.githubusercontent.com/1804568/100765226-8c606b80-33f7-11eb-826d-4a8de22a689d.png">